.eslintrc.js

```diff
@@ -16,6 +16,41 @@ module.exports = {
                 ' * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause',
                 ' '
             ]
+        ],
+        // Sometimes VS Code will automatically add missing imports, but will do so via
+        // @salesforce/retail-react-app/node_modules/ instead of a normal import. It technically
+        // works inside the monorepo, but won't necessarily work in customer projects, so we want to
+        // avoid the pattern. (Also just because it's weird.)
+        'no-restricted-imports': [
+            'error',
+            {
+                patterns: [
+                    {
+                        group: ['@salesforce/retail-react-app/node_modules/*'],
+                        message: 'Did your IDE auto-complete this import? Avoid /node_modules/ and import the package directly.'
```

Comment:
I use webstorm, and it does not import with node_modules, it knows to import directly from the package